custom_components/tuya_local/devices/ion_wifi_dimmer.yaml

```diff
@@ -0,0 +1,58 @@
+name: ION WIFI DIMMER
```

Comment:
The top level name should be generic (without branding).
"Dimmer" would seem to be suitable.

Reply:
Renamed to "Dimmer".